validator/src/main/resources/expected-data-template/python/eks/rds-mysql-trace.mustache

```diff
@@ -0,0 +1,94 @@
+[
+  {
+    "name": "^{{serviceName}}$",
+    "origin": "^AWS::EKS::Container$",
+    "subsegments": [
+      {
+        "name": "^mysql$",
+        "namespace": "^remote$",
+        "aws": {
+          "account_id": "^{{accountId}}$",
+          "xray": {
+            "auto_instrumentation": true,
+            "sdk": "^opentelemetry for python$"
+          }
+        },
+        "sql": {
+          "connection_string": "^localhost/information_schema$",
+          "url": "^SELECT$",
+          "sanitized_query": "^SELECT \\* FROM tables LIMIT 1;$",
+          "database_type": "^mysql$"
+        },
+        "metadata": {
+          "default": {
+            "aws.span.kind": "^CLIENT$",
+            "EKS.Cluster": "^e2e-python-canary-test$",
+            "PlatformType": "^AWS::EKS$"
+          }
+        },
+        "annotations": {
+          "aws.local.service": "^{{serviceName}}$",
+          "aws.local.operation": "^GET mysql$",
+          "aws.remote.resource.type": "^DB::Connection$",
+          "aws.remote.resource.identifier": "^{{remoteResourceIdentifier}}$",
+          "aws.remote.service": "^mysql$",
+          "aws.remote.operation": "^SELECT$"
+        }
+      }
+    ],
+    "http": {
+      "request": {
+        "url": "^{{endpoint}}/mysql$",
+        "method": "^GET$"
+      },
+      "response": {
+        "status": 200
+      }
+    },
+    "aws": {
+      "account_id": "^{{accountId}}$",
+      "xray": {
+        "auto_instrumentation": true,
+        "sdk": "^opentelemetry for python$"
+      }
+    },
+    "metadata": {
+      "default": {
+        "otel.resource.aws.log.group.names": "^/aws/containerinsights/e2e-python-canary-test/application$",
+        "otel.resource.service.name": "^{{serviceName}}$",
+        "aws.span.kind": "^LOCAL_ROOT$",
+        "otel.resource.cloud.platform": "^aws_eks$",
+        "EKS.Cluster": "^e2e-python-canary-test$",
+        "otel.resource.telemetry.sdk.name": "^opentelemetry$",
+        "otel.resource.ec2.tag.kubernetes.io/cluster/e2e-python-canary-test": "^owned$",
+        "otel.resource.k8s.container.name": "^back-end$",
+        "otel.resource.cloud.account.id": "^{{accountId}}$",
+        "http.route": "^mysql$",
+        "PlatformType": "^AWS::EKS$",
+        "otel.resource.telemetry.sdk.language": "^python$",
+        "otel.resource.cloud.provider": "^aws$"
+      }
+    },
+    "annotations": {
+      "aws.local.service": "^{{serviceName}}$",
+      "aws.local.operation": "^GET mysql$"
+    }
+  },
+  {
+    "name": "^mysql$",
+    "origin": "^Database::SQL$",
+    "inferred": true,
+    "sql": {
+      "connection_string": "^localhost/information_schema$",
```

Comment:
May I know why is this localhost?

Reply:
The [Java instrumentation repository](https://github.com/aws-observability/aws-otel-java-instrumentation), creates a span with the connection string in it, and it appears under the `db.connection_string` field in the span. The Python instrumentation repository doesn't actually propagate such field. I've tried to extract the spans for pymysql and psycopg2, and I couldn't find that field within the span. This behavior is coming from the upstream repository, not from the [aws-otel-python-instrumentation repository](https://github.com/aws-observability/aws-otel-python-instrumentation/).